pypeit/core/coadd.py

```diff
@@ -2578,7 +2578,7 @@ def ech_combspec(waves, fluxes, ivars, masks, weights_sens, nbest=None, wave_met
     #                          header=header, ex_value=ex_value, overwrite=True)
 
 
-    return (wave_giant_stack, flux_giant_stack, ivar_giant_stack, mask_giant_stack), \
+    return wave_grid_mid, (wave_giant_stack, flux_giant_stack, ivar_giant_stack, mask_giant_stack), \
```

Comment:
Please update the docs to reflect the new return value.

Reply:
Done.